umh.docs.umh.app/content/en/docs/features/postgresql_and_grafana_alerts.md

```diff
@@ -0,0 +1,245 @@
+        the results of your queries and functions by clicking on **Preview**
+        and check if they function correctly and fire an alert.
+     
+     3. Section three defines the rule location, the time interval for rule 
+        checking, and the duration for which the rule has to be broken before
+        an alert is triggered.
+        - Select a name for your rule's folder or add it to an existing 
+          one by clicking the arrow. Find all your rules grouped in these
+          folders on the **Alert rules** page under **Alerting**.
+        - An **Evaluation group** is a grouping of rules, which are checked 
+          after the same time interval. Creating a new group requires setting a
+          time interval for rule checking. The minimum interval
+          from Grafana is ten seconds.
+        - Specify the duration the rule must be violated before triggering the
+          alert. For example, with a ten-second check interval and a 20-second
+          duration, the rule must be broken twice in a row before an alert is
+          fired.
+   
+     4. In section four, you can add details and descriptions for your rule.
+     
+     5. In section five, you are required to assign labels to your alert, 
+        ensuring it is directed to the appropriate contacts. For example,
+        you may designate a label **team** with **alertrule1**: `team = operator`
+        and **alertrule2:** `team = management`. It can be helpful to use
+        labels more than once, like  **alertrule3:** `team = operator`, to link 
+        multiple alerts to a contact point at once.
+     
+     Your rule is now completed; click on **Save and Exit** on the right upper 
+     corner, next to section one.
+
+3. **Create a Contact Point**
+
+     A contact point consolidates addresses and services to be notified. When a
+     linked alert is triggered, everyone within the contact point receives a 
+     message. To create a contact point, navigate to **Contact points**, located
+     at the top of the Grafana alerting page.
+     - Click on the blue **+ Add contact point** button.
+     - Choose a name for your contact point.
+     - Pick the receiving service; in this example, Discord.
+     - Generate a new Webhook in your Discord server (Server Settings → 
+       Integrations → View Webhooks → New Webhook or create Webhook). 
+       Assign a name to the Webhook and designate the messaging channel.
+       Copy the Webhook URL from Discord and insert it into the corresponding 
+       field in Grafana. Customize the message to Discord under 
+       **Optional Discord settings** if desired.
+     - If needed, add more services to the contact point, by clicking **+ Add 
+       contact point integration**.
+     - Save the contact point; you can see it in the **Contact points** list,
+       below the **grafana-default-email** contact point.
+   
+   The integration of alerting by mail is very complicated to set up in Grafana
```

Comment:
is it really very complicated? i would say the setup of a mail server is complicated, to add it to Grafana its easy again

Reply:
yes, you need many not so commonly known information of your provider, there is not a easy way to access these. The many different providers like google, microsoft.... add to the complexity.
or in Ferdinants words: "are you shure you want to add it?"